dependency/setup_ipmi-exporter.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+if [ ! -s /etc/pos/pos-prometheus.yml ]
+then
+    echo "Skiping IPMI-Exporter setup"
+    exit 1
+fi
+
+pid=$(sudo lsof -t -i:9290)
+
+if [ ! -z "$pid" ]
+then
+    sudo kill $pid
+    echo "Process with PID $pid using port 9290 has been killed"
+fi
+
+wget https://github.com/prometheus-community/ipmi_exporter/releases/download/v1.6.1/ipmi_exporter-1.6.1.linux-amd64.tar.gz
```

Comment:
Can we keep a local copy copy of IPMI exporter for offline installation, for systems which does not have github access?

Reply:
Yes, we should keep local copy since github access might not be there in each server

Reply:
Thank you for the comments. Updated script file please check.